src/pages/Home.js

```diff
@@ -16,11 +16,51 @@ export default function Home(props) {
     history.push('/list');
   }
 
+  function handleInputValue(e) {
+    setInputValue(e.target.value.trim());
+  }
+
+  function checkExistingToken(e) {
+    e.preventDefault();
+    if (!inputValue) {
+      return swal('Please enter your token!', 'Input is empty', 'error');
+    }
+    db.collection(inputValue)
+      .get()
+      .then((snap) => {
+        if (snap.empty) {
+          swal(
+            'Token not found!',
+            'Please try again or start a new list!',
+            'error',
+          );
+        } else {
+          addKeyValuePairToLocalStorage('token', inputValue);
+          props.setToken(inputValue);
+          history.push('/list');
+        }
+      });
```

Comment:
Great job with using sweet alert! So satisfying!! I love how you handled both empty string AND faulty input!

Reply:
🎉